app/src/main/java/mozilla/lockbox/action/RouteAction.kt

```diff
@@ -16,4 +16,5 @@ sealed class RouteAction : Action {
     object Back : RouteAction()
     object LockScreen : RouteAction()
     data class ItemDetail(val id: String) : RouteAction()
+    data class OpenApp(val host: String) : RouteAction()
```

Comment:
Naming: Is this `OpenApp` or `OpenWebsite`

Reply:
do you mean to have 2 entries one for apps and one for websites ?

Reply:
In my opinion, if the method is called `openWebsite()`, then the Action ought to be called `OpenWebsite` instead of `OpenApp`.

Reply:
sorry about confusion, i was thinking that at some point this open feature will open websites and installed apps. I had updated action name as suggested